src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/SimpleRenameStrategy.cs

```diff
@@ -19,11 +19,11 @@ public SimpleRenameStrategy(IProjectThreadingService threadingService, IUserNoti
         }
 
         // For the SimpleRename, it can attempt to handle any situtation
-        public override bool CanHandleRename(string oldFileName, string newFileName)
+        public override bool CanHandleRename(string oldFileName, string newFileName, bool isCaseSensitive)
         {
             var oldNameBase = Path.GetFileNameWithoutExtension(oldFileName);
             var newNameBase = Path.GetFileNameWithoutExtension(newFileName);
-            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase);
+            return SyntaxFacts.IsValidIdentifier(oldNameBase) && SyntaxFacts.IsValidIdentifier(newNameBase) && (string.Compare(Path.GetFileName(oldNameBase), Path.GetFileName(newNameBase), isCaseSensitive) !=0);
```

Comment:
Can you add tests cases for this? If I rename encyclopedia to encyclopædia it should still rename.